azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADAuthenticationFilter.java

```diff
@@ -36,16 +36,21 @@
 
     private static final String TOKEN_HEADER = "Authorization";
     private static final String TOKEN_TYPE = "Bearer ";
+    private static final String AUTH_URL_HEADER = "AuthUrl";
 
     private AADAuthenticationFilterProperties aadAuthFilterProp;
+    private ServiceEndpointsProperties serviceEndpointsProp;
 
-    public AADAuthenticationFilter(AADAuthenticationFilterProperties aadAuthFilterProp) {
+    public AADAuthenticationFilter(AADAuthenticationFilterProperties aadAuthFilterProp,
+                                   ServiceEndpointsProperties serviceEndpointsProp) {
         this.aadAuthFilterProp = aadAuthFilterProp;
+        this.serviceEndpointsProp = serviceEndpointsProp;
```

Comment:
please help to update readme.md to mention that azure cn is also supported, and how to use it.

Reply:
README is updated. And now one property is added on server side to configure the azure service instance value.